src/app/profile/settings/settings.component.less

```diff
@@ -0,0 +1,45 @@
+@import (reference) '../../../assets/stylesheets/shared/osio.less';
+
+.settings-nav {
+  position: relative;
+  overflow: hidden;
+}
+
+#settings-title-nav {
+  background-color: #393F44;
+  color: #ffffff;
+  height: 60px;
+  padding: 18px 15px 15px 25px;
+}
+
+#settings-title-nav h2, h3 {
```

Comment:
You should use a class, rather than ID's and subsequent targeting of HTML elements.